src/Components/SearchForm.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '../Styles/SearchForm.css';
+
+class SearchForm extends React.PureComponent {
+
+    state = {
+        inputValue: '',
+        showSearchTerm: false
+    };
+
+    updateInputValue = (event) => {
+        this.setState({
+            inputValue: event.target.value,
+            showSearchTerm: false
+        });
+        console.log(this.state.inputValue)
+    };
+
+    handleSubmit = (evt) => {
+        evt.preventDefault();
+        this.setState({
+            showSearchTerm: true
+        });
+    };
+
+    render() {
+        return (
+            <div>
+                <h2>{this.props.title}</h2>
+                <form>
+                    <input type="text" value={this.state.inputValue} onChange={this.updateInputValue} placeholder="What do you want to watch?" />
+                    <button onClick={this.handleSubmit}>Search</button>
+                </form>
+                {this.state.showSearchTerm === true ? <h3>you searched for {this.state.inputValue}</h3> : null}
```

Comment:
In most React guides you will find next code styling:
```
{this.state.showSearchTerm && <h3>...</h3>}
```

This is slightly shorter, just don't be surprised if you see it. `<h3>` will be rendered only if first condition is truthy.